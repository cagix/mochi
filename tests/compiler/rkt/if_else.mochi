let x = 2
if x == 1 {
  print(1)
} else if x == 2 {
  print(2)
} else {
  print(3)
}
