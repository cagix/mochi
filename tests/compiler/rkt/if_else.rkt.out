#lang racket

(define x 2)
(if (= x 1)
	(begin
		(displayln 1)
	)
	(if (= x 2)
		(begin
			(displayln 2)
		)
		(begin
			(displayln 3)
		)
	)
)

